src/test/java/uk/gov/hmcts/dm/service/AuditedStoredDocumentOperationsServiceTests.java

```diff
@@ -187,5 +189,18 @@ public void testBulkUpdateDocumentWithError() {
         when(storedDocumentService.findOne(storedDocument.getId())).thenReturn(Optional.empty());
         auditedStoredDocumentOperationsService.updateDocument(storedDocument.getId(), metadata, newTtl);
     }
+
+    @Test
+    public void testDeleteCaseStoredDocumentsShouldMarkAllDocumentsForDeletion() {
+        List<StoredDocument> storedDocuments = Stream.of(TestUtil.STORED_DOCUMENT).collect(Collectors.toList());
+        storedDocuments.addAll(Stream.of(TestUtil.STORED_DOCUMENT).collect(Collectors.toList()));
+
+        CaseDocumentsDeletionResults caseDocumentsDeletionResults =
+            auditedStoredDocumentOperationsService.deleteCaseStoredDocuments(storedDocuments);
+
+        assertThat(caseDocumentsDeletionResults.getCaseDocumentsFound().equals(storedDocuments.size()));
+        assertThat(caseDocumentsDeletionResults.getMarkedForDeletion().equals(storedDocuments.size()));
+    }
+
 }
```

Comment:
Can we have one more Test where the CaseDocumentsFound don't match the MarkedForDeletion?

Reply:
Unlikely to happen in real life situation, but added a test for this scenario.